app/jobs/process_pull_request_update_job.rb

```diff
@@ -0,0 +1,7 @@
+class ProcessPullRequestUpdateJob < ApplicationJob
+  queue_as :default
+
+  def perform(action, author, url, html_url)
```

Comment:
```suggestion
  def perform(action, github_username, url, html_url)
```

I think we should name this consistently inside for us. It's unclear to me whether "author" is a string, object, DB record, etc.